.travis.yml

```diff
@@ -11,6 +11,20 @@ script:
 - go vet ./...
 - go test -v -race ./...
 
+after_script:
+- cd /home/travis/gopath/src; mkdir k8s.io; cd k8s.io
+- git clone https://github.com/kubernetes/kubernetes.git; cd kubernetes; git checkout release-1.11
+- GO111MODULE=auto make WHAT=cmd/kube-apiserver
+- go mod init
+- cd /home/travis/gopath/src/k8s.io/kubernetes/staging/src/k8s.io/apiserver; go mod init
```

Comment:
possible to use ENV for GOPATH: /home/travis/gopath/src?

Reply:
Yep, updated it.